### PR TITLE
Cleanup of the magnetogram/SRS example

### PR DESCRIPTION
**Pull request by ayshih**
Follow-on to #5719

### CODE REVIEW

**Review by dstansby — DISMISSED** [review]
This is broken in the doc build: https://sunpy--5741.org.readthedocs.build/en/5741/generated/gallery/plotting/magnetogram_active_regions.html#sphx-glr-generated-gallery-plotting-magnetogram-active-regions-py

**Comment by ayshih** [comment]
Wha...?  Kicking this to draft

**Comment by ayshih** [comment]
Ah, of course, my change is limited to Astropy 5.0.  Hmm...

**Comment by nabobalis** [comment]
Docfail is 
```
/home/vsts/work/1/s/docs/generated/gallery/plotting/magnetogram_active_regions.rst:71: WARNING: py:meth reference target not found: astropy.visualization.wcsexes.WCSAxes.plot_coord
```
Might need to be skipped in https://github.com/sunpy/sunpy/blob/main/docs/nitpick-exceptions

File: examples/plotting/magnetogram_active_regions.py [hunk 1]
```diff
@@ -30,9 +30,17 @@
 
 ##############################################################################
 # Now we extract the latitudes, longitudes and the region numbers.
+# :meth:`~astropy.visualization.wcsexes.WCSAxes.plot_coord` will error on some
```
Comment (dstansby) [inline examples/plotting/magnetogram_active_regions.py:33]:
```suggestion
# :meth:`~astropy.visualization.wcsaxes.WCSAxes.plot_coord` will error on some
```

**Comment by nabobalis** [comment]
Ah, I didn't spot the typo.

**Review by nabobalis — APPROVED** [review]

**Pull request merged**